hack/e2e.sh

```diff
@@ -57,6 +57,17 @@ Examples:
 
     See https://onsi.github.io/ginkgo/ for more ginkgo options.
 
+3) reuse the cluster and don't tear down it after the testing
+
+    REUSE_CLUSTER=y SKIP_DOWN=y ./hack/e2e.sh -- <e2e args>
+
+4) use registry mirrors
+
+    DOCKER_IO_MIRROR=https://dockerhub.azk8s.cn ./hack/e2e.sh -- <e2e args>
```

Comment:
```suggestion
    DOCKER_IO_MIRROR=https://dockerhub.azk8s.cn QUAY_IO_MIRROR=quay.azk8s.cn GCR_IO_MIRROR=gcr.azk8s.cn ./hack/e2e.sh -- <e2e args>
```

add `QUAY_IO_MIRROR=quay.azk8s.cn` and `GCR_IO_MIRROR=gcr.azk8s.cn` to this example?

Reply:
done